src/styles/_catalog.scss

```diff
@@ -14,9 +14,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-  }
+    transition-duration: 300ms;
```

Comment:
Use the short syntax of transition and don't forget to add a parameter which will use this transition